### ISSUE TITLE
Metrics/ClassLength cop failing on multi-variable assignment after 0.90.0

### ISSUE THREAD

**Issue by boomer196** [rubocop-hq/rubocop#8880]
## Expected behavior

Metrics/ClassLength cop should be able to examine the file

## Actual behavior

```
An error occurred while Metrics/ClassLength cop was inspecting

undefined method 'class_definition?' for :X:Symbol
undefined method 'class_definition?' for :Y:Symbol
```

## Steps to reproduce the problem

The line of code it was failing on was similar to this:
```
class Foo
  A = "blah".freeze
  X = Y = Z = [["x", "X"], ["y", "Y"], ["z", "Z"]].freeze
end
```

I was upgrading from `0.90.0` to `0.93.0` when I found the issue.

## RuboCop version
```
$ [bundle exec] rubocop -V
0.93.0 (using Parser 2.7.2.0, rubocop-ast 0.7.1, running on ruby 2.6.6 x86_64-darwin18)
```

**Comment by natematykiewicz** [comment]
Are you intentionally assigning 3 constants to the same array of arrays? I agree that this should parse, but it also seems like you meant to write:

```ruby
X, Y, Z = [["x", "X"], ["y", "Y"], ["z", "Z"]].freeze
```

Example:

```ruby
A = "blah".freeze
A
=> "blah"


X = Y = Z = [["x", "X"], ["y", "Y"], ["z", "Z"]].freeze
X
=> [["x", "X"], ["y", "Y"], ["z", "Z"]]
Y
=> [["x", "X"], ["y", "Y"], ["z", "Z"]]
Z
=> [["x", "X"], ["y", "Y"], ["z", "Z"]]
```

I guess I'm also trying to confirm that your code example is accurate to the real-world problem.

**Comment by boomer196** [comment]
Yes, it's intentional.  I probably should have used a different example.  I apologize.  Thanks for following up.

This should cause it to fail as well:

`A = B = C = "blah".freeze`

vs this and the cop can process:

```
A = "blah".freeze
B = A
C = A
```

**Comment by natematykiewicz** [comment]
@koic is very fast! #8880

**Comment by key88sf** [comment]
I have the same problem. Thanks for fixing!
